api/cluster.go

```diff
@@ -27,3 +29,66 @@ func (s *Server) appStatus(ctx *middleware.Context) {
 
 	response.Write(ctx, response.NewError(http.StatusServiceUnavailable, errors.New("node not ready")))
 }
+
+func (s *Server) getClusterStatus(ctx *middleware.Context) {
+	status := models.ClusterStatus{
+		Node:  cluster.ThisNode,
+		Peers: cluster.GetPeers(),
+	}
+	response.Write(ctx, response.NewJson(200, status, ""))
+}
+
+var NotFoundErr = errors.New("not found")
```

Comment:
declarations for global variables should be after the imports (imho or best practice not sure)